scripts/main.js

```diff
@@ -71,4 +71,59 @@ const departments = {
     }
 }
 
-console.log(departments);
\ No newline at end of file
+console.log(departments);
+
+/*Opdracht 1a*/
+console.log("De afdeling Sales heeft " + departments.sales.numberOfEmployees + " medewerkers");
+
+/*Opdracht 1b*/
+console.log("Marketing is een leuke afdeling om te werken. " + departments.marketing.description);
+
+/*Opdracht 1c*/
+console.log("De afdeling Customer Service heeft " + departments["customer-service"].numberOfEmployees + " medewerkers");
+
+/*Opdracht 1d*/
+console.log("Sales is een uitdagende afdeling om te werken als Verkoopmanager. " + departments.sales.jobs[1].description);
+
+/*Opdracht 2a*/
+const departmentInput = prompt ("Over welke afdeling wil je meer informatie? Kies uit: [marketing / sales / customer-service]");
+console.log(departmentInput);
+
+/*Opdracht 2b en 2c*/
+if (departmentInput === "marketing") {
+    console.log("Je koos Marketing. " + departments.marketing.description);}
+else if (departmentInput === "sales") {
+    console.log("Je koos Sales. " + departments.sales.description);}
+else if (departmentInput === "customer-service") {
+console.log("Je koos Customer-Service. " + departments["customer-service"].description);}
+else {
+    console.error("Ongeldige keuze. Probeer het opnieuw door de pagina te verversen.");}
```

Comment:
Het principe van 4b is een beetje als dit
```suggestion
switch(departmentInput){
    case "marketing":
    case "sales":
    case "customer-service":
        console.log(`Je koos ${departmentInput}. ${departments[departmentInput].description}`);
        break;
    default:
        console.error("Ongeldige keuze. Probeer het opnieuw door de pagina te verversen.");
        break;
}
```

Wellicht dat je dit principe kan toepassen voor opdracht 4b en 4c 😉

Reply:
Anders kun je nog spieken bij de uitwerkingen:
https://github.com/hogeschoolnovi/frontend-javascript-job-prompter-uitwerkingen

Reply:
Dankjewel voor het nakijken. Het is duidelijk :-)